src/main/java/com/google/devtools/build/lib/runtime/commands/DumpCommand.java

```diff
@@ -445,7 +445,8 @@ private void dumpSkylarkHeap(BlazeWorkspace workspace, String path, PrintStream
     if (allocationTracker == null) {
       out.println(
           "Cannot dump skylark heap without running in memory tracking mode. "
-              + "Please refer to the user manual for the dump commnd "
+              + "Please refer to "
+              + "https://docs.bazel.build/versions/master/command-line-reference.html#dump "
```

Comment:
Can you add "For bazel please refer to " so our internal users don't go there? Internally we use a different java agent, so I wouldn't want to point them to this.